File: rb/utility/make_context.rb
# RemoteJobsApiRssFeed SDK utility: make_context
require_relative '../core/context'
module RemoteJobsApiRssFeedUtilities
  MakeContext = ->(ctxmap, basectx) {
    RemoteJobsApiRssFeedContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// RemoteJobsApiRssFeed SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class RemoteJobsApiRssFeedMakeContext
{
    public static function call(array $ctxmap, ?RemoteJobsApiRssFeedContext $basectx): RemoteJobsApiRssFeedContext
    {
        return new RemoteJobsApiRssFeedContext($ctxmap, $basectx);
    }
}
